napari/layers/utils/interactivity_utils.py

```diff
@@ -116,15 +116,17 @@ def orient_plane_normal_around_cursor(layer: Image, plane_normal: tuple):
     from napari.layers.image._image_constants import VolumeDepiction
 
     viewer = napari.viewer.current_viewer()
+    if viewer is None:
+        return
 
     # early exit
     if viewer.dims.ndisplay != 3 or layer.depiction != VolumeDepiction.PLANE:
         return
 
     # find cursor-plane intersection in data coordinates
     cursor_position = layer._world_to_displayed_data(
-        position=viewer.cursor.position,
-        dims_displayed=layer._slice_input.displayed,
+        position=np.asarray(viewer.cursor.position),
+        dims_displayed=np.asarray(layer._slice_input.displayed),
```

Comment:
I believe this should stay a list as well:

```
 def _world_to_displayed_data(
        self, position: np.ndarray, dims_displayed: List[int]
    ) -> npt.NDArray:
```

Reply:
And actually 

```
   37     @property
   38     def displayed(self) -> List[int]:
   39         """The layer dimension indices displayed in this slice."""
   40         return list(self.order[-self.ndisplay :])
```

so the original should have been fine.

```suggestion
        dims_displayed=layer._slice_input.displayed,
```